RUFAS/routines/field/crop/root_development.py

```diff
@@ -29,7 +30,7 @@ def __init__(self, crop_data: Optional[CropData] = None):
         # data reference
         self.data = crop_data or CropData()  # defaults if not given
```

Comment:
```suggestion
        self.data = crop_data or CropData()
```
We should be able to delete these inline comments.